Maui.RevenueCat.InAppBilling/Platforms/Android/Extensions/EntitlementInfosExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Com.Revenuecat.Purchases;
+using Maui.RevenueCat.InAppBilling.Extensions;
+using Maui.RevenueCat.InAppBilling.Models;
+
+namespace Maui.RevenueCat.InAppBilling.Platforms.Android.Extensions;
+internal static class EntitlementInfosExtensions
+{
+    internal static List<EntitlementInfoDto> ToEntitlementInfoDtoList(this EntitlementInfos entitlements)
+    {
+        var entitlementInfoDtos = new List<EntitlementInfoDto>();
```

Comment:
Here please rename local variable `entitlements` as `entitlementInfos` so it properly reflects the type.
I prefer this approach (especially for parameters) because otherwise it can sometimes bring confusion that there are 2 different types, in this case `EntitlementInfos` and `Entitlements`

- same for iOS